src/create_disk.sh

```diff
@@ -207,22 +242,23 @@ if [ "${save_var_subdirs}" != NONE ]; then
 	chcon -h $(matchpathcon -n /home) ${vardir}/home
 fi
 
-if [ "$arch" == "x86_64" ]; then
-	# install bios grub
-	grub2-install \
-		--target i386-pc \
-		--boot-directory rootfs/boot \
-		$disk
-	ext="X64"
-elif [ "$arch" == "aarch64" ]; then
-	mkdir -p rootfs/boot/grub2
-	ext="AA64"
-fi
-
 # we use pure BLS, so don't need grub2-mkconfig
 ostree config --repo rootfs/ostree/repo set sysroot.bootloader none
 
-if [ "$arch" != "s390x" ]; then
+case "$arch" in
+x86_64|aarch64)
+    if [ "$arch" == "x86_64" ]; then
```

Comment:
lets just have separate cases for these two arches, especially now that we already have a case statement

Reply:
Sure, will split out aarch64 completely.